cfgov/agreements/templates/agreements/index.html

```diff
@@ -0,0 +1,246 @@
+{% endblock %}
+
+{% block title %}Credit card agreement database{% endblock %}
+
+{% block breadcrumbs %}
+<li><a href="/">Home</a></li>
+<li><a href="/credit-cards/">Credit cards</a></li>
+<li><span>Credit card agreements</span></li>
+{% endblock %}
+
+{% block content %}
+<h1>Credit card agreement database</h1>
+
+<div id="ccagrhome">
+		<p>
+        The CFPB maintains a database of credit card agreements from hundreds of card issuers.
+            Using the tool below, you can search for an agreement by the name of the issuer.
+		</p>
+
+	  <h5>Show agreements by card issuer</h5>
+    {% issuer_select %}
+
+    <a href="#modIssuer" class="popl">Can't find an issuer?</a>
+    <div id="modIssuer" class="modal closed" style="display: none;">
+        <a href="#modIssuer" class="close">Close</a>
+
+        <span><a href="#">Can't find an issuer?</a></span>
+        <p>
+            The drop-down list contains credit card issuers that have submitted
+            credit card agreements as required under Section 204 of the
+            Credit Card Accountability Responsibility and Disclosure Act of 2009
+            (the CARD Act).
+        </p>
+        <p>
+            Look on the back of the credit card or on your latest monthly statement
+            to find the name of the issuer. The bank, retailer, or other entity
+            whose name appears on the front of the card may not be
+            the actual issuer of the card. You can also check the issuer's website;
+            typically, the bank name is listed at the bottom of the issuer's homepage.
+        </p>
+        <p>
+            If you cannot find an agreement, call your issuer to request
+            a copy of the agreement. Federal law requires your issuer to
+            give you a copy of your agreement on request.
+        </p>
+    </div>
+
+    <p>
+        This search method accesses all credit card agreements as of the CFPB’s most recent quarterly collection of
+        credit card agreements. To access all agreements from the most recent or earlier periods, please click on the
+        relevant agreement link below. (Please note that the Bureau did not collect agreements for 2015. The January
+        2016 archive is a sample that was collected by Bureau staff from the publicly-available websites of the largest
+        credit card issuers as of the first week of January 2016, and does not constitute a full set of agreements).
+        Credit card issuers are generally required to post the credit card agreements that they offer to the public on
+        their websites, with limited exceptions.
+        If you are an issuer, email <a href="mailto:CardAgreements@consumerfinance.gov">CardAgreements@consumerfinance.gov</a>
+        for agreement submission instructions.
+    </p>
+
+
+    <p id="dwnld">
+        <a href="https://s3.amazonaws.com/files.consumerfinance.gov/a/assets/Credit_Card_Agreements_2017_Q2.zip">
```

Comment:
It'd be nice if these weren't hardcoded and there wasn't so much duplication. Do we have any template tag that could be used for these? Perhaps it'd be better for a separate PR, but we could have a tag that provided this prefix under `agreements/templatetags`.

Reply:
I was trying *not* to revist application code at this point-- that's a good idea for a seperate PR.